bin/templates/scripts/cordova/lib/Podfile.js

```diff
@@ -78,7 +78,7 @@ Podfile.prototype.__parseForDeclarations = function (text) {
 
     // getting lines between "platform :ios, '11.0'"" and "target 'HelloCordova'" do
     const declarationsPreRE = new RegExp('platform :ios,\\s+\'[^\']+\'');
-    const declarationsPostRE = new RegExp('target\\s+\'[^\']+\'\\s+do');
+    const declarationsPostRE = new RegExp('target\\s+\'.+\'\\s+do');
```

Comment:
This reminded me of something. 

The `.` does not grab whitespaces, whereas the previous one (`[^\']`) did.

Can you confirm if this doesn't break apps with spaces in the name? Or simply correct my wrong assumptions if those are the case 😅

Reply:
`.` grabs whitespaces, it doesn't grab newlines IIRC.

I tested with spaces and it works.

Reply:
I stand corrected. It grabs ` ` indeed, missing new lines. Does it grabs other kind of spacing like \t ? 🤔

Reply:
I'm pretty sure it captures everything other than line separation characters. Those can be captured by the exact characters and `\s`.